bootcamp-milestone-2/src/app/components/blogPreview.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import style from "./blogPreview.module.css";
+import Image from "next/image";
+import { Blog } from "../blogData";
+export default function BlogPreview(props: Blog) {
+  return (
+    // replace everything between the <div> & </div> tags
+    // with your code from earlier milestones
+    <div className={style.blog}>
+      <div>
+        <h3>{props.title}</h3>
+        <Image src={props.image} alt="img" width={400} height={300}></Image>
+        <p>{props.description}</p>
+        <p>{props.date}</p>
```

Comment:
Where are your links to an individual blog page? Blog previews are not that useful if you can't actually read your blog! For this milestone, you can just hard code pages. Later on we will make this dynamic